src/main/java/servlets/Calculator.java

```diff
@@ -0,0 +1,138 @@
+        for (int i = 0; i < modequation.length; i++)
+            if ('0' <= modequation[i] && modequation[i] <= '9') {
+                int j = 0;
+                if (!stack.isEmpty() && prev == 1) {
+                    if (stack.peek().matches("[-+]?\\d+")) {
+                        j = Integer.parseInt(stack.peek().trim());
+                        stack.pop();
+                        stack.push(j + String.valueOf(modequation[i]));
+                        prev = 1;
+                    }
+                } else {
+                    stack.push(String.valueOf(modequation[i]));
+                    prev = 1;
+                }
+            } else {
+                if (modequation[i] == '+' || modequation[i] == '-') {
+                    if (!nstack.isEmpty()) {
+                        if (nstack.peek().equals("+") || nstack.peek().equals("-") ||
+                                nstack.peek().equals("*") || nstack.peek().equals("/")) {
+                            stack.push(String.valueOf(nstack.peek()));
+                            nstack.pop();
+                        }
+                    }
+                    nstack.push(String.valueOf(modequation[i]));
+                    prev = 0;
+                }
+                if (modequation[i] == '*' || modequation[i] == '/') {
+                    if (!nstack.isEmpty()) {
+                        if (nstack.peek().equals("*") || nstack.peek().equals("/")) {
+                            stack.push(String.valueOf(nstack.peek()));
+                            nstack.pop();
+                        }
+                    }
+                    nstack.push(String.valueOf(modequation[i]));
+                    prev = 0;
+                }
+                if (modequation[i] == '(') {
+                    nstack.push("(");
+                    prev = 0;
+                }
+                if (modequation[i] == ')') {
+                    prev = 0;
+                    do {
+                        stack.push(String.valueOf(nstack.peek()));
+                        nstack.pop();
+                    } while (!nstack.peek().equals("("));
+                    nstack.pop();
+                }
+            }
+        while (!stack.isEmpty()) {
+            nstack.push(stack.peek());
+            stack.pop();
+        }
+        int leftoperand = 0;
+        while (!nstack.isEmpty()) {
+        leftoperand = Integer.parseInt(nstack.peek().trim());
+        stack.push(nstack.peek());
+        nstack.pop();
+        int rightoperand;
+        if (!nstack.isEmpty())
+            rightoperand = Integer.parseInt(nstack.peek().trim());
+        else break;
+        stack.push(nstack.peek());
+        nstack.pop();
+        while (nstack.peek().matches("[-+]?\\d+")) {
+            leftoperand = rightoperand;
+            rightoperand = Integer.parseInt(nstack.peek().trim());
+            stack.push(nstack.peek());
+            nstack.pop();
+        }
+        switch (nstack.peek()) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4500113)